src/components/icon/IconSize.ts

```diff
@@ -1,4 +1,5 @@
 export enum IconSize {
+  VERY_SMALL = 'very_small',
```

Comment:
I think we dont need this. We dont need to follow 100 % the design and have our icon size as everywhere else it was used. SMALL size :)